modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -823,7 +823,7 @@ public void setParameterExampleValue(CodegenParameter p) {
             if (example == null) {
                 example = p.paramName + "_example";
             }
-            example = "\"" + escapeText(example) + "\"";
+            example = escapeText(example);
```

Comment:
Is this a must? What issues are you trying to address? I think this will impact many other Java-related generators so you may consider overriding the function in the spring generators instead.